src/editors/containers/EditorContainer/hooks.js

```diff
@@ -11,11 +11,40 @@ export const {
   saveBlock,
 } = appHooks;
 
+export const setAssetToStaticUrl = (images, getContent) => {
```

Comment:
Can you add a comment here for future readers which summarizes the conversion? I think it would be something like:
`// For assets to remain usable across course instances, we convert their url to be course-agnostic. That for example, /course/assets/<asset hash>/ filename` gets converted to '/static/<etc>/<fileName>'

Reply:
Also might be worth to note that if we add an assets storage method for libraries, this would be something that would need to choose? I don't really know how far off assets are in libraries, but good to be future-proof.